### PR TITLE
Fix styling issues with rendering example code

### PR DESCRIPTION
**Pull request by Zain-Abbas1**
__Short description outlining key changes/additions__
- Fix styling issue, background appears as intended along with correct format of code.
- No GDS for this, have to use custom css for the moment.
- Fix broken link to authentication page on Getting started page.

__JIRA Ticket Number__
FA-220

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by mgearon-ch** [comment]
@ch-dclarke @Zain-Abbas1 With the font size issue, we could change it from 14px to 0.9rem to use the % percentage value of the html root level but then that would have the same issue John is having with the sidebar being too small as the CDN CSS is wrong at the moment as the HTML font-size is set to 62% rather than 100%.

Also the CSS will most likely change as the project develops so I think it would passing it for the moment?

File: src/main/resources/templates/dev-hub/authentication.html [hunk 1]
```diff
@@ -47,20 +47,17 @@ <h3 class="govuk-heading-m">HTTP Basic Authentication</h3>
             demonstrates the setting of the
             <code>Authorization</code>
             HTTP request header, as defined by <a
-                href="https://www.example.com/" class="govuk-link">RFC2617</a>.
+                href="https://tools.ietf.org/html/rfc2617" class="govuk-link">RFC2617</a>.
         </p>
         <!-- https://tools.ietf.org/html/rfc2617 sticking this link here for easy retrival if we need if when updating the content of this page. -->
```
[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply src/main/resources/templates/dev-hub/authentication.html]: Done

**Comment by CH-JonathanBlake** [comment]
It sounds like a bad idea to code for it to look right now accounting for that error, unless we've given up on resolving that bug.
Otherwise we will eventually fix it (hopefully before it hits live) and then we'll have to fix this css anyway.

**Comment by Zain-Abbas1** [comment]
I've changed values to use `rem`, although this does not match what is currently there I agree that a single fix for the sidebar should also resolve this error. Based upon that I'm willing to push it through with the `rem` values in the css. However, for consistency we must look at all the other `px` values currently in the application.css file as well.

File: src/main/resources/static/css/application.css [hunk 2]
```diff
@@ -163,4 +163,177 @@
...
+@media (min-width: 641px) {
+  pre,
+  code,
+  .code,
+  pre code,
+  pre code .attribute,
+  table td.resource,
+  table td code,
+  table span.value {
+    font-size: 14px;
```
[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply src/main/resources/static/css/application.css]: Done

File: src/main/resources/static/css/application.css [hunk 3]
```diff
@@ -163,4 +163,177 @@
...
+pre code,
+pre code .attribute,
+table td.resource,
+table td code,
+table span.value {
+  color: #008000;
+  font-family: "proxima_nova", Arial, sans-serif;
+  font-weight: 400;
+  text-transform: none;
+  font-size: 12px;
```
[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply src/main/resources/static/css/application.css]: Done

[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply src/main/resources/static/css/application.css]: Duplicated comment - see above.

File: src/main/resources/static/css/application.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zain-Abbas1) [reply src/main/resources/static/css/application.css]: Duplicated comment - see above.

**Pull request merged**